Scripts/Insights_19.10.3_hotfix.ps1

```diff
@@ -0,0 +1,52 @@
+﻿param(
+   [Parameter(Mandatory=$true)][string]$username
+  ,[Parameter(Mandatory=$true)][string]$password
+)
+
+function Get-checkBuild($tenant)
+{
+        $cubeBuilding = 0
+        while ($cubeBuilding -ne 1)
+        {
+            $buildResult = & "C:\Program Files (x86)\UiPath\Orchestrator\Tools\UiPath.InsightsAdminTool.exe" buildStatus -t $tenant | out-string
+            $buildStatus = $buildResult.Split(“`n”)
+
+            if($buildStatus[1].StartsWith("Build successfully ended"))
+            {
+                $cubeBuilding = 1;
+            }
+            else
+            {
+                Write-Output "$tenant Cube is still building. Waiting..."
+                Start-Sleep -s 10
+            }
+        }
+
+}
+
+Write-Output "Copying .jar"
+Copy-Item test.txt -Destination "C:\Program Files\Sisense\DataConnectors\JVMContainer\Connectors\UiFrost\com.sisense.connectors.jdbc.UiFrost.jar" -Recurse -force
```

Comment:
`test.txt`?

Reply:
Right, let's make sure our file is not test.txt
Also what happens if the file is in use at the time of the copy, do we need to stop a service first?

Reply:
Old value before I got the .jar from Hal. I got a new copy of Hal's .jar and validated the file with the new timestamp was moved to the uIFrost folder :

 Copy-Item com.sisense.connectors.jdbc.UiFrost.jar -Destination "$uifrostPath\com.sisense.connectors.jdbc.UiFrost.jar" -Recurse -force

The .jar was replaced without issue. The -force command is supposed to copy an item that can not otherwise be changed. I have not seen any issue replacing this file while Sisense is running. 

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/copy-item?view=powershell-7#parameters